Resources/Prototypes/_White/Entities/Effects/effects.yml

```
@@ -36,4 +36,24 @@
       netsync: false
       drawdepth: Effects
     - type: PointLight
-      color: "#008DFE"
\ No newline at end of file
+      color: "#008DFE"
+
+- type: entity
+  id: EffectTeslaSparksSilent
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: TimedDespawn
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)